backend/src/v2/cmd/driver/main.go

```diff
@@ -86,6 +86,10 @@ var (
 	noProxy           = flag.String(noProxyArg, unsetProxyArgValue, "Addresses that should ignore the proxy.")
 	publishLogs       = flag.String("publish_logs", "true", "Whether to publish component logs to the object store")
 	cacheDisabledFlag = flag.Bool("cache_disabled", false, "Disable cache globally.")
+
+	mlPipelineServiceTLSEnabled = flag.Bool("ml_pipeline_service_tls_enabled", false, "Set to true if mlpipeline api server serves over TLS (default: false).")
+	metadataTLSEnabled          = flag.Bool("metadata_tls_enabled", false, "Set to true if metadata server serves over TLS (default: false).")
```

Comment:
I think the default should show in the --help.
```suggestion
	metadataTLSEnabled          = flag.Bool("metadata_tls_enabled", false, "Set to true if metadata server serves over TLS.")
```